### PR TITLE
Airflow modifications of the pgbouncer exporter

### PR DESCRIPTION
**Pull request by potiuk**
Pgbouncer image as needed by Apache Airflow

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -1,9 +1,40 @@
...
+ARG PGBOUNCER_EXPORTER_VERSION
+ARG AIRFLOW_PGBOUNCER_EXPORTER_VERSION
+ARG GO_VERSION
+
+LABEL org.apache.airflow.component="pgbouncer-exporter"
+LABEL org.apache.airflow.pgbouncer_exporter.version="${PGBOUNCER_EXPORTER_VERSION}"
+LABEL org.apache.airflow.go.version="${GO_VERSION}"
+LABEL org.apache.airflow.airflow_pgbouncer_exporter.version="${AIRFLOW_PGBOUNCER_EXPORTER_VERSION}"
+
+MAINTAINER "Apache Airflow Community <devlist@airflow.apache.org>"
```
Comment (turbaszek) [inline Dockerfile]:
```suggestion
MAINTAINER "Apache Airflow Community <dev@airflow.apache.org>"
```
Or I miss something?

File: Dockerfile [hunk 2]
```diff
@@ -1,9 +1,40 @@
...
+ARG PGBOUNCER_EXPORTER_VERSION
+ARG AIRFLOW_PGBOUNCER_EXPORTER_VERSION
+ARG GO_VERSION
+
+LABEL org.apache.airflow.component="pgbouncer-exporter"
+LABEL org.apache.airflow.pgbouncer_exporter.version="${PGBOUNCER_EXPORTER_VERSION}"
+LABEL org.apache.airflow.go.version="${GO_VERSION}"
+LABEL org.apache.airflow.airflow_pgbouncer_exporter.version="${AIRFLOW_PGBOUNCER_EXPORTER_VERSION}"
+
+MAINTAINER "Apache Airflow Community <dev@airflow.apache.org>"
```
Comment (turbaszek) [inline Dockerfile:30]: Out of curiosity, can anyone send an email to the dev list?

Comment (potiuk) [reply Dockerfile:30]: Yes

**Review by turbaszek — APPROVED** [review]

**Comment by potiuk** [comment]
Added Commit SHA as label as well to keep full track of image origin

File: Dockerfile (hunk 2 above)
Comment (ashb) [reply Dockerfile:30]: (It'll get moderated/held in a queue, but yes)

**Pull request merged**